src/app/services/baw-api/baw-api.service.ts

```diff
@@ -284,13 +284,22 @@ export class BawApiService<
       map(this.handleSingleResponse(classBuilder))
     );
 
+    if (model["image"] === "") {
+      const removeImageReqBody = `{ "${model.kind.toLowerCase()}": { "image": null } }`;
+      return request.pipe(
+        mergeMap(() => this.httpPatch(path, removeImageReqBody)),
+        map(this.handleSingleResponse(classBuilder))
+      );
+    }
+
```

Comment:
OK; multiple things wrong with this:

- this change is in the wrong spot (it should be withing `hasFormDataOnlyAttributes` and `hasFormDataOnlyAttributes`)
- you shouldn't be using empty string to denote and empty value. `"" !== null`.
- this will only work for fields named `image`
   - the true bug here is with how we serialize fields that must be sent for form-data requests.